core/query/evalNestedQuery.ml

```diff
@@ -30,8 +30,10 @@ let tag_query : QL.t -> QL.t =
         | Erase (e, fields) -> Erase (tag e, fields)
         | Variant (l, e) -> Variant (l, tag e)
         | XML v -> XML v
-        | Apply (f, es) -> Apply (f, List.map tag es)
+        | Apply (u, vs) -> Apply (tag u, List.map tag vs)
+(*        | ApplyPrim (f, es) -> ApplyPrim (f, List.map tag es) *)
```

Comment:
Unsure why this comment is here.  Is this something we might want to add, or a reminder that we don't need this here?

Reply:
Certainly not needed any more. Looks like a reminder of the previous state of affairs I left when changing the code.